### DISCUSSION TITLE
Zotac Nvidia RTX 3060 Twin Edge LHR 12Gb GDDR6  -lhr-mode 2 -lhr-mode 1

### DISCUSSION THREAD

**Discussion by scorpions333z** [NebuTech/NBMiner#465] [Q&A]
Use RaveOs Driver 1.0.25 (1983)
nvidia-460.84, -lhr-mode 2 totally improves the temperature from 64g to 54g but it goes to 28.16 hashrate without modifying anything, in lhr-mode 1 it goes to 29.45 hash but with higher temperatures, which I recommend to put in oc - Core Clock offset, MHz? -Memory Clock offset, MHz? - Power limit, W?

**Comment by JDDelgado** [comment]
hola, yo manejo power limit 80, core clock en -502, memory clock x1615, y obtengo de 36 a 37mhs estables, temperatura de 57g con buena ventilacion 98w de consumo con esa 3060 lhr v2
